### DISCUSSION TITLE
reference/getStarted/install

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#890] [Q&A]
# reference/getStarted/install

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.devlive.org/reference/getStarted/install.html

**Comment by yangzhongye** [comment]
import io.edurt.datacap.sql.parser.SqlBaseLexer;
import io.edurt.datacap.sql.parser.SqlBaseParser;
请问拉的源码里缺少这两个类呢？

**Reply by qianmoQ** [reply]
需要使用 `./mvnw -T 1C clean install package -Dspotbugs.skip -Dgpg.skip -Dcheckstyle.skip -DskipTests=true` 命令编译一下，才会出现这些类
